geode-core/src/main/java/org/apache/geode/cache/client/internal/RegisterInterestTracker.java

```diff
@@ -106,7 +108,8 @@ public static int getInterestLookupIndex(boolean isDurable, boolean receiveUpdat
     return result;
   }
 
-  void addSingleInterest(final @NotNull LocalRegion r, final @NotNull Object key,
+  @VisibleForTesting
+  public void addSingleInterest(final @NotNull LocalRegion r, final @NotNull Object key,
```

Comment:
Can the test needing this method be moved into the same package to retain the package-private?

Reply:
I am not sure of the benefit. Can you explain that? I understand the basic security benefit of having package private, but using the visible for testing will make something private. What is the difference you are hoping to preserve? The availability to the rest of the package? If I delete the public, won't that do the trick?

As far as moving this to somewhere else it seems like I would be making encapsulation worse. I am open to it.